api/src/main/java/org/asynchttpclient/Request.java

```diff
@@ -112,6 +113,13 @@
     InputStream getStreamData();
 
     /**
+     * Return the current request's body as a Reactive Stream Publisher.
+     *
+     * @return a Reactive Stream Publisher representation of the current request's body.
```

Comment:
This will return null if no publisher was set, right?  That should probably be documented, we definitely don't want to expose any arbitrarily set body as a publisher as implementing that will require either depending on a reactive streams implementation from the async-http-client API, or implementing reactive streams ourselves.